### PR TITLE
deployment: SONAR instance

### PR DESCRIPTION
**Pull request by sebastiendeleze**
* NEW environment variable for API URL
* FIX Change static directory mapping
* FIX Change web server domain

Signedoff-by: Sébastien Délèze <sebastien.deleze@rero.ch>

### CODE REVIEW

**Comment by jma** [comment]
change the commit message: this commit is not only for dev. Capitalise the title and add dots to items.

**Comment by sebastiendeleze** [comment]
I just have a question about merging. Actually master is ahead from this pull request. 

In you opinion, is it better to merge master into pull request's branch before close pull request or simply merge the pull request ? 
I think the first is better because I can resolve eventually conflicts before merging to master and merging can be fast-forwarded. But maybe I miss something ? 

Thanks for your help

File: setup.py [hunk 1]
```diff
@@ -55,6 +55,7 @@
         ],
         'invenio_config.module': [
             'sonar = sonar.config',
+            'sonar-specific = sonar.modules.config'
```
Comment (jma) [inline setup.py]: why not sonar-modules?

File: sonar/modules/config.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+#
+# Copyright (C) 2019 RERO.
+#
+# Swiss Open Access Repository is free software; you can redistribute it
+# and/or modify it under the terms of the MIT License; see LICENSE file for
+# more details.
+
+"""SONAR specific configuration."""
+
+SONAR_FIXTURES_API_URL = 'https://127.0.0.1:5000/api/'
```
Comment (jma) [inline sonar/modules/config.py]: why not localhost?

File: sonar/modules/ext.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# My site is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+"""Author extension."""
```
Comment (jma) [inline sonar/modules/ext.py]: change it

File: sonar/modules/config.py (hunk 2 above)
Comment (sebastiendeleze) [reply sonar/modules/config.py]: I think 127.0.0.1 always exists, it's not the case for localhost. But, as discussed, we keep localhost for consistency with invenio

**Pull request merged**